docs/docs/04-standard-library/01-cloud/api.md

```diff
@@ -160,6 +161,31 @@ new cloud.Api(props?: ApiProps);
 
 ---
 
+##### `bind` <a name="bind" id="@winglang/sdk.cloud.Api.bind"></a>
```

Comment:
Can we hide this from the docs? It should be possible to add it to the list of hidden methods here: https://github.com/winglang/wing/blob/8df5d0e87549e61105710c1a0a18191d1abb2179/apps/jsii-docgen/src/docgen/view/wing-filters.ts#L10

The method isn't meant to be directly called in Wing applications, so I think it might be more confusing to have it shown here.